scripts/env_matrix.yml

```diff
@@ -4,6 +4,7 @@ CONDA_PY:
   - "3.5"
 CONDA_BOOST: "1.60"
 CONDA_PERL: "5.22.0"
+iCONDA_R: "3.3.0"
```

Comment:
This is basically also a test how easy it is to update R. If it goes flawlessly, I'll try to follow the R channel more closely.